### DISCUSSION TITLE
how to enable "oh my posh" like icons?

### DISCUSSION THREAD

**Discussion by madsholme** [Eugeny/tabby#9420] [Q&A]
I can see a screenshot where it is in the tabby page, but doing a ssh from my windows to my linux server, they do not show up. I dont seem to find any where to enable them in the config? or have i overlooked it?

**Comment by Eugeny** [comment]
From the sound of it you might want to try a Powerline-patched font

**Comment by madsholme** [comment]
So in other words install oh my posh?

________________________________
